tests/Feature/DealExposeControllerTest.php

```
@@ -0,0 +1,417 @@
+            'name' => 'Default Agent',
+            'email' => 'agent'.uniqid('', true).'@test.com',
+            'status' => 'active',
+            'is_client_contact' => null,
+            'image' => null,
+            'mobile' => null,
+            'salutation' => null,
+            'created_at' => now(),
+            'updated_at' => now(),
+        ]);
+
+        return User::withoutGlobalScopes()->findOrFail($id);
```

PHPStan (2.2.8)

[error] Method Tests\Feature\DealExposeControllerTest::createAgent() should return App\Models\User but returns array<App\Models\User|Illuminate\Database\Eloquent\Builder>|Illuminate\Database\Eloquent\Builder|Illuminate\Database\Eloquent\Collection|Illuminate\Database\Eloquent\Model.


(return.type)